integration/up/hybridmode_test.go

```diff
@@ -49,9 +61,25 @@ dev:
 
 	svcDockerfile = `FROM busybox
 ENV ENV_IN_IMAGE value_from_image`
+	envFile      = `TEST_ENV_FILE_VAR1=from-file-1`
 	localProcess = `
 #!/bin/bash
 
+if [ "$#" -ne 0 ]; then
```

Comment:
this check is not particularly useful now, but it doesn't harm. It was initially helpful because as part of this PR I added the support for `args` which I moved to a separate  issue and I would like to re-use this check in the future once we introduce args.